android/res/values-cs/strings.xml

```diff
@@ -526,7 +526,7 @@
 	<!-- Cuisine -->
 	<string name="cuisine_chinese">čínská kuchyně</string>
 	<!-- Cuisine -->
-	<string name="cuisine_coffee_shop">kavárna</string>
+	<string name="cuisine_coffee_shop">Káva</string>
```

Comment:
Это не важно по двум причинам:
1. код все равно делает заглавные. Всегда.
2. В стрингах переводы остались виеменно, только для андроида. Они должны боаться из json файлов через ядро. Дима пока не добрался до этого изменения.

> On 18.3.2016, at 15.32, Ilya Zverev <notifications@github.com> wrote:
> 
> In android/res/values-cs/strings.xml:
> 
> > @@ -526,7 +526,7 @@
> >  	<!-- Cuisine -->
> >  	<string name="cuisine_chinese">čínská kuchyně</string>
> >  	<!-- Cuisine -->
> > -	<string name="cuisine_coffee_shop">kavárna</string>
> > +	<string name="cuisine_coffee_shop">Káva</string>
> Почему здесь с заглавной, когда соседние — со строчной?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>